.gitlab-ci.yml

```diff
@@ -171,6 +156,8 @@ create-instances:
   stage: run-instances
   dependencies:
     - create-instances
+  rules:
+    - if: '$CI_PIPELINE_SOURCE == "push"'
```

Comment:
shouldn't this include `if: '$NIGHTLY == "true"'` too?

Reply:
No, it should be included in server master only

Reply:
my bad